scheduler/src/cook/kubernetes/api.clj

```diff
@@ -464,8 +470,14 @@
                   (log/info "In" compute-cluster-name "compute cluster, initializing node watch helper")
                   (initialize-node-watch-helper compute-cluster)
                   (catch Exception e
-                    (log/error e "Error during node watch initial setup of looking at nodes for" compute-cluster-name
-                               "and sleeping" reconnect-delay-ms "milliseconds before reconnect")
+                    (let [state @state-atom]
+                      (if (= state :deleted)
+                        (log/info
+                          e "In" compute-cluster-name "compute cluster, cannot create node watch (cluster is deleted)"
```

Comment:
can we still leave the message saying that there's a sleep?

Reply:
👍🏼 Done